public/index.html

```diff
@@ -15,8 +15,8 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" />
     <link href="%REACT_APP_GOOGLE_FONTS_LINK%" rel="stylesheet" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <!-- fathom analytics -->
-    <script src="https://cdn.usefathom.com/script.js" data-site="%REACT_APP_FATHOM_DATA_SITE%" defer></script>
+    <!-- analytics -->
+    <script defer data-domain="%REACT_APP_DOMAIN_URL%,%REACT_APP_AGGREGATED_DOMAIN_URL%"src="https://plausible.io/js/plausible.local.js"></script>
```

Comment:
This is still pointing to local, so won't work when deployed. As discussed this should be changed to `https://plausible.io/js/plausible.js`